pom.xml

```
@@ -54,7 +54,8 @@ THE SOFTWARE.
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/jenkins-clone-workspace-scm-plugin</gitHubRepo>
-    <jenkins.version>2.346.3</jenkins.version>
+<!--    <jenkins.version>2.346.3</jenkins.version>-->
+    <jenkins.version>2.361.1</jenkins.version>
```

This change also needs the changes described in ["Update Jenkins version"](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/).  The bom entry will need to change form `bom-2.346.x` to `bom-2.361.x`.  Based on ["Update Jenkins version"](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/#update-minimum-required-jenkins-version)  it should require 2.361.4 rather than 2.361.1.
```suggestion
    <jenkins.version>2.361.4</jenkins.version>
```